yt_idefix/fields.py

```diff
@@ -3,8 +3,8 @@
 
 
 class IdefixVtkFieldInfo(FieldInfoContainer):
-    known_other_fields = (  # type: ignore
-        ("RHO", ("code_mass / code_length**3", ["density"], None)),
+    known_other_fields = (
+        ("RHO", ("code_mass / code_length**3", ["density"], None)),  # type: ignore
```

Comment:
Probably equivalent, slightly more elegant in my opinion
```suggestion
    known_other_fields = (   # type: ignore
        ("RHO", ("code_mass / code_length**3", ["density"], None)),
```

Reply:
Unfortunately, your original location of ` # type: ignore` couldn't pass `mypy` in my machine.

Reply:
ok fair enough, thanks for checking !